src/client/src/ui/blotter/model/blotterModel.js

```diff
@@ -46,6 +47,10 @@ export default class BlotterModel extends ModelBase {
     this._schedulerService = schedulerService;
   }
 
+  get canPopout() {
+    return Environment.isRunningInIE;
```

Comment:
good god I only noticed this. It was there before my PR and I blindly lifted it from the middle of the file to near the top.

I had a quick search and looks like this prop is in a few place. It's very odd indeed.

@fdore does it ring any bells?